src/backend/api/Fusion.Resources.Api/Controllers/Departments/ApiModels/ApiDepartmentResponsible.cs

```diff
@@ -22,6 +23,24 @@ public ApiDepartmentResponsible(QueryDepartmentResponsible responsible)
                 CreatedBy = new ApiPerson(responsible.CreatedBy);
         }
 
+        public ApiDepartmentResponsible(QueryDepartmentResponsible responsible, bool limit)
```

Comment:
What does limit do? It hides the reason and CreatedBy, why?

Reply:
Good question. It should maybe be commented in the method, but the reason for why we limit it is related to this:
https://statoil-proview.visualstudio.com/787035c2-8cf2-4d73-a83e-bb0e6d937eec/_workitems/edit/50694